modular_zubbers/code/modules/antagonists/modglue.dm

```diff
@@ -43,14 +45,16 @@
 		alert = tgui_alert(stripper,
 			"As you go to pull the modsuit off of [owner], you spot a what appears to be a failsafe. \n If you pull any further, it looks like a fuse for a bomb will be pulled. Continue???",
 			"Are you REALLY SURE?",
-			list("Yes","No"), 10 SECONDS, TRUE)
+			list(PULL_SUIT, IGNORE_SUIT, DUMP_CONTENTS), 10 SECONDS, TRUE)
 
 
-	if(alert == "Yes")
+	if(alert == PULL_SUIT)
 		ASYNC
 			if(dna_lock.dna)
 				dna_lock.explode(user = stripper, owner = owner)
 		. = ..()
+	if(alert == DUMP_CONTENTS)
+		dump_modsuit_toss_contents()
```

Comment:
???? Why are you doing this??? I thought the whole point was them not being loot pinatas??? I legitimately do not understand where you're going with this PR. Why are you making the DNA lock EMP resistant _**JUST**_ to make it so pulling out the suit dumps it's contents???

Reply:
Just give them a goddamn microbomb at this point.

Reply:
I don't think you understand the point. Based on feedback, players wanted to be able to get stolen things back from contractors.  They wanted to be able to retreive AI's and gear - This doesn't dump the modules. They also didn't like the macrobomb, so now we get a microbomb that blows up the suit (after dumping its contents).

Reply:
There will be a dump contents added to empty the modsuit storage as per my requirement for this PR.

Reply:
> There will be a dump contents added to empty the modsuit storage as per my requirement for this PR.

this defeats the entire point of having the suit self destruct- it doesn't actually stop them being a loot pinata in anyway except removing the baton from the round

Reply:
I'd also say that retrieving stolen items is a big reason not to instantly kill contractors- why not just shoot them to take the items off their corpse if you have 0 reason to take them in alive. There are very few items that are truly irreplacable and almost none of them are round critical, there's the very niche cases of the contractor lugging brains or AI cards in their bag but as per the RR and the nature of their objectives there's very little reason they'd do either.-- To put it simpler, if the contractor happens to steal the X0-1 earlier in the round it gives the HoS an incentive to detain them with non lethals/not let them die whereas with the current system there's no reason not to just shoot them to death on the spot and strip their suit for the free antag goodies---- **INCLUDING A PINPOINTER THAT IGNORES SUIT SENSORS**